### PR TITLE
Feature/1612 flerbindspost titel

### PR DESCRIPTION
**Pull request by JrgenGNielsen**
https://platform.dandigbib.org/issues/1612

### CODE REVIEW

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: b1197daba14796303038ac17556d224365ad5156
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

 * Commit: 405449552b01e1994b6469e721f692c51972ab6e
  * The subject should be kept around 50 characters long. The subject must not be more than 69 characters long. Strive for about 50 characters.

 * Commit: e1ce854f694bbf6aac86ede117dbcb8d778ec95c
  * The first line of your commit message (the subject) must follow this format: [issue number]: [short summary of change]

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: 3034bc6b3cbde661ff327c5b02aaed7cb30542d3
  * Your commit message body contains a line that is longer than 72 characters

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Please ensure that your commit messages comply with our contribution guidelines.

 * Commit: 3034bc6b3cbde661ff327c5b02aaed7cb30542d3
  * Your commit message body contains a line that is longer than 72 characters

Guidelines are available at https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md.

---
This message was auto-generated by https://gitcop.com

File: modules/ting/ting.field.inc [hunk 1]
```diff
@@ -363,8 +363,18 @@ function ting_field_formatter_view($entity_type, $entity, $field, $instance, $la
...
+          foreach ($collection->entities as $collection_entity) {
+            if ($collection_entity_title == NULL) {
+              $collection_entity_title = $collection_entity->title;
+            }
+            elseif ($collection_entity_title != $collection_entity->title) {
+              // Replace title with the more generic collection title.
+              $title = $collection->title;
+              $ting_type = t('Material collection');
+              break;
+            }
```
Comment (kasperg) [inline modules/ting/ting.field.inc]: I have tried reading through this loop + conditionals multiple times and I am still having trouble wrapping my head around the code here.

After having read the issue again then how about changing it to something like this:

```php
$collection_titles = array_map(function($collection_entity) {
  return $collection_entity->title;
});
// If a collection has items of the same type but with different titles then use the generic collection title
if (count(array_unique($collection_titles) > 1) {
    $title = $collection->title;
    $ting_type = t('Material collection');
}
```

**Note: The code is completely untested!**

**Pull request merged**